bindings/python/opendal.pyi

```diff
@@ -33,6 +34,10 @@ class AsyncOperator:
     async def create_dir(self, path: str): ...
     async def delete(self, path: str): ...
 
+class BlockingReader:
```

Comment:
Since we are utilizing both `Operator` and `AsyncOperator`, perhaps it would be beneficial to adopt a similar naming convention as that of `Reader` and `AsyncReader`?

Reply:
https://github.com/datafuselabs/opendal/blob/main/src/types/reader.rs#L179

Doesn't it look good the way it is now?

Reply:
Python and Rust have different programming paradigms. I am aligning with existing Python libraries to create a module that functions like a native Python module.

Reply:
You are right. I searched for it and get it.